tests/functional/aws-node-sdk/test/bucket/deleteBucketTagging.js

```diff
@@ -0,0 +1,94 @@
+const assert = require('assert');
+const { errors } = require('arsenal');
+const { S3 } = require('aws-sdk');
+const async = require('async');
+
+const getConfig = require('../support/config');
+
+const bucket = 'policyputtaggingtestbucket';
+
+const validTagging = {
+    TagSet: [
+        {
+            Key: 'key1',
+            Value: 'value1',
+        },
+        {
+            Key: 'key2',
+            Value: 'value2',
+        },
+    ],
+};
+
+// Check for the expected error response code and status code.
+function assertError(err, expectedErr, cb) {
+    if (expectedErr === null) {
+        assert.strictEqual(err, null, `expected no error but got '${err}'`);
+    } else {
+        assert.strictEqual(err.code, expectedErr, 'incorrect error response ' +
+            `code: should be '${expectedErr}' but got '${err.code}'`);
+        assert.strictEqual(err.statusCode, errors[expectedErr].code,
+            'incorrect error status code: should be  ' +
+            `${errors[expectedErr].code}, but got '${err.statusCode}'`);
+    }
+    cb();
+}
+
+describe.only('aws-sdk test delete bucket tagging', () => {
+    let s3;
+
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        return done();
+    });
+
+    beforeEach(done => s3.createBucket({ Bucket: bucket }, done));
+
+    afterEach(done => s3.deleteBucket({ Bucket: bucket }, done));
+
+    it('should delete tag', done => {
+        async.series([
+            next => s3.putBucketTagging({
+                AccountId: s3.AccountId,
+                Tagging: validTagging, Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+            next => s3.getBucketTagging({
+                AccountId: s3.AccountId,
+                Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+            next => s3.deleteBucketTagging({
+                AccountId: s3.AccountId,
+                Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+            next => s3.getBucketTagging({
+                AccountId: s3.AccountId,
+                Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+        ], (err) => {
+            assertError(err, 'NoSuchTagSet', done);
+        });
+    });
```

Comment:
I don't understand what is tested here, but I think we should check at each step that the operation returned the right results, i.e. that the first GET returns something, and the second returns an empty array.

Reply:
I probably should've been clearer in the PRs notes: this PR's tests are unfinished until I can get this change through https://github.com/scality/Arsenal/pull/1814